### ISSUE TITLE
Wrong usage of ANSI terminal colors

### ISSUE THREAD

**Issue by peterpp** [nette/tester#417]
Version: 1.7.2, 2.0.2, 2.3.1

By default, tester output in terminal is colored. But those colors are used wrongly.

First of all, "bright" version of colors can be interpreted as bold text with "basic" color. So color "gray" as defined in Dumper.php can be displayed as bold black. Therefore you can't use "gray" to look texts with less importance.

Than you can't use "white" as font text because with standard ANSI colors and white terminal background, these texts will be barely visible. "white" is usable only together with colored background.

I played with output and this is my suggestion to fix it (all in Dumper.php):
- replace `($item['file'] === $testFile ? self::color('white') : '')` with `($item['file'] === $testFile ? self::color('navy') : '')` to highlight file path with row number
- replace ` self::color('white', $message)` with `self::color(null, $message)`
- replace `self::color('white')` with `self::color()`
- replace `self::color('gray')` with `self::color()`

**Comment by dg** [comment]
Do you meet terminals that understand bright as bold?

**Comment by peterpp** [comment]
Of course. Standard terminal on macOS (and also terminal integrated in PhpStorm).

Standard terminal:
![Screen Shot 2020-01-02 at 21 38 52](https://user-images.githubusercontent.com/1448883/71692095-421ad880-2da9-11ea-8b6e-5c41ef506ddc.png)

Standard terminal with "Use bright colors for bold text" - the result is even worse and practically not usable:
![Screen Shot 2020-01-02 at 21 39 45](https://user-images.githubusercontent.com/1448883/71692141-64145b00-2da9-11ea-9eff-33c80786cdc4.png)

Colors turned off in terminal – nearly everything is bold, not ideal:
![Screen Shot 2020-01-02 at 21 50 26](https://user-images.githubusercontent.com/1448883/71692364-f0bf1900-2da9-11ea-876c-7c78f76876d7.png)

With my changes and bold instead of bright:
![Screen Shot 2020-01-02 at 21 43 22](https://user-images.githubusercontent.com/1448883/71692181-84dcb080-2da9-11ea-9ba6-61af502bccbc.png)

With my changes and colors off:
![Screen Shot 2020-01-02 at 21 51 32](https://user-images.githubusercontent.com/1448883/71692452-31b72d80-2daa-11ea-94ce-8873eb66bc22.png)

Finally, yellow color for displaying values is perfect with dark terminal theme, but not very readable with white background. However, I prefer using bold font instead of bright colors.
